soh/soh/Enhancements/randomizer/randomizer.cpp

```diff
@@ -4209,7 +4307,28 @@ CustomMessageEntry Randomizer::GetMerchantMessage(RandomizerInf randomizerInf, u
     return messageEntry;
 }
 
-const char* mapGetItemHints[3][2] = {
+CustomMessageEntry Randomizer::GetSshMessage(s16 params) {
```

Comment:
Same thing here with naming, I get it's the actor name but this is one of the actor names that are _really_ bad and aren't apparent at first glance